src/completion.ts

```diff
@@ -360,6 +373,24 @@ class DictionaryCompletionItemProvider implements vscode.CompletionItemProvider
                     return this.completeByFirstLetter(firstLetter, addSpace);
                 }
                 return [];
+            // Julia
+            case "julia":
+                //// Multiline comment
+                if (/\#\=((?!\=\#)[\W\w])*$/.test(docTextBefore)) {
+                    return this.completeByFirstLetter(firstLetter, addSpace);
+                }
+                //// Inline comment or string
+                const tmpTextBeforeJulia = textBefore.replace(/(?<!\#)('|").*?(?<!\#)\1/g, '');
+                if (
+                    /\#{1,}/.test(tmpTextBeforeJulia) //// inline comment
+                    || (
+                        /(?<!\#)['"]/.test(tmpTextBeforeJulia) //// inline string
+                        && !/(import|require)/.test(tmpTextBeforeJulia.split(/['"]/)[0]) //// reject if in import/require clauses
```

Comment:
Doesn't seem right. I guess it should be something about `import`, `using`, `include` for Julia?